### PR TITLE
feat: API completion - Swagger Implementation (#40314)

### PR DESCRIPTION
**Pull request by nx-mukesh**
swagger for trainee pending.

### CODE REVIEW

**Comment by nx-mukesh** [comment]
All API working robust with security and permissions.

File: src/controller/trainee/routes.ts [hunk 1]
```diff
@@ -8,35 +8,207 @@ import validation from './validation';
...
+// delete swagger -
+/**
+ * @swagger
+ * /trainee/:id:
+ *   delete:
+ *     description: Delete trainee
+ *     tags: [Trainee]
+ *     parameters:
+ *         - in: path
+ *           name: id
```
Comment (namanpar) [inline src/controller/trainee/routes.ts]: wright swagger description and then route. DO not wright them separately.

File: src/controller/user/routes.ts [hunk 2]
```diff
@@ -7,41 +7,236 @@ import { TRAINER, USER } from '../../libs/constants';
...
+ *         schema:
+ *           properties:
+ *            _id:
+ *              type: string
+ *            originalId:
+ *              type: string
+ *            name:
+ *              type: string
+ *            email:
+ *              type: string
```
Comment (namanpar) [inline src/controller/user/routes.ts]: same as above

**Review by namanpar — CHANGES_REQUESTED** [review]

File: src/controller/user/routes.ts (hunk 2 above)
Comment (nx-mukesh) [reply src/controller/user/routes.ts]: fixed

File: src/controller/trainee/routes.ts (hunk 1 above)
Comment (nx-mukesh) [reply src/controller/trainee/routes.ts]: fixed

**Review by namanpar — APPROVED** [review]

**Pull request merged**